packages/quantic/force-app/solutionExamples/main/lwc/exampleSearch/e2e/exampleSearch.e2e.ts

```diff
@@ -0,0 +1,151 @@
+      searchPage,
+      search,
+    }) => {
+      // Trigger a search query
+      const exampleQuery = 'test';
+      const searchRequestPromise = search.waitForSearchRequest();
+
+      await searchPage.searchboxInput.fill(exampleQuery);
+      await searchPage.searchboxInput.press('Enter');
+
+      await searchRequestPromise;
+      const searchRequestBody = (await searchRequestPromise).postDataJSON();
+      expect(searchRequestBody?.q).toBe(exampleQuery);
+
+      await expect(searchPage.summary).toContainText(exampleQuery);
+
+      const recentQueriesListItems =
+        await searchPage.recentQueriesListItems.all();
+      expect(recentQueriesListItems.length).toEqual(1);
+      const expectedQueries = [exampleQuery];
+      expectedQueries.forEach(async (expectedQuery: string, index: number) => {
+        // eslint-disable-next-line no-await-in-loop
+        await expect(recentQueriesListItems[index]).toHaveText(expectedQuery);
+      });
+
+      // Change the sort order
+      const sortClickSearchRequestPromise = search.waitForSearchRequest();
+      await searchPage.sortDropdownButton.click();
+      const sortLabelWithValue = await searchPage.getSortLabelValue();
+      const {label: expectedSortName, value: expectedSortValue} =
+        sortLabelWithValue[1];
+
+      await searchPage.clickSortButton(expectedSortName);
+      const searchRequestBodyAfterSortClick = (
+        await sortClickSearchRequestPromise
+      ).postDataJSON();
+      expect(searchRequestBodyAfterSortClick?.sortCriteria).toBe(
+        expectedSortValue
+      );
+
+      // Change the result page
+      const pagerClickSearchRequestPromise = search.waitForSearchRequest();
+      await searchPage.clickOnPagerButtonByIndex(2);
+
+      const searchRequestBodyAfterPagerClick = (
+        await pagerClickSearchRequestPromise
+      ).postDataJSON();
+
+      expect(searchRequestBodyAfterPagerClick?.firstResult).toBe(10);
+
+      // Change the tab
+      const expectedTabName = 'Articles';
+      const tabClickSearchRequestPromise = search.waitForSearchRequest();
+
+      const secondTab = await searchPage.getTabByIndex(1);
+      await secondTab.click();
+
+      const searchRequestBodyAfterTabClick = (
+        await tabClickSearchRequestPromise
+      ).postDataJSON();
+      expect(searchRequestBodyAfterTabClick?.tab).toBe(expectedTabName);
+
+      // Select a facet value
+      const facetSelectSearchRequestPromise = search.waitForSearchRequest();
+      await searchPage.clickOnFirstFacetsContainer();
+      const firstFacetValue = await searchPage.getFacetValueByIndex(0);
+      await firstFacetValue.click({force: true});
+
+      expect(firstFacetValue).not.toBeNull();
+      const searchRequestBodyAfterFacetSelect = (
+        await facetSelectSearchRequestPromise
+      ).postDataJSON();
+
+      const selectedFacetMatchesExpectedFacet =
```

Comment:
This too, go read the text value of the first facet value, and then click on it, then expect that.